src/main/java/org/timinggame/api/room/controller/RoomController.java

```diff
@@ -23,4 +23,10 @@ public ResponseEntity<CreateRoomRes> startGame(@PathVariable("roomId") @Positive
         Room room = roomService.startGame(roomId);
         return ResponseEntity.ok(CreateRoomRes.builder().roomId(room.getRoomId()).build());
     }
+
+    @PostMapping("/{pinCode}/enter")
+    public ResponseEntity<CreateRoomRes> enterPinCode(@PathVariable("pinCode") String pinCode) {
```

Comment:
```suggestion
    public ResponseEntity<CreateRoomRes> enterPinCode(@PathVariable("pinCode") final String pinCode) {
```

final 붙여주세요
반환 타입도 수정해야 될 것 같아요